src/test/java/org/apache/commons/lang3/RandomUtilsTest.java

```diff
@@ -93,15 +92,6 @@ public void testNextFloatLowerGreaterUpper() {
         assertThrows(IllegalArgumentException.class, () -> RandomUtils.nextFloat(2, 1));
     }
 
-    /**
-     * Tests next boolean
-     */
-    @Test
-    public void testBoolean() {
```

Comment:
This test does nothing. `RandomUtils.nextBoolean()` returns a `boolean` that can either be `true` or `false`. This is guaranteed by the language, there is no need to test it. It is kind of evident.